### ISSUE TITLE
Upon Adding Gun Plugin Geyser is not working

### ISSUE THREAD

**Issue by shivpatel333** [GeyserMC/Geyser#2449]
### Describe the bug

When I added Quality Armory Plugin to my Server then BedRock Players were not able to join the Server.

### To Reproduce

-

### Expected behaviour

I exepect that you will fix this bug as soon as possible, so that our Bedrock Players can use guns

### Geyser Version

-

### Minecraft: Bedrock Edition Version

1.17.1

**Comment by TheMonBonGH** [comment]
we can't "fix this" if we can't reproduce this, which you have provided no info.

**Comment by shivpatel333** [comment]
I didn't add anything on that because I am not a Coder or Developer. So I dont know how to solve that. I described the Issue to you!

**Comment by Jens-Co** [comment]
What TheMonBon means that you need to upload logs (latest.log) with this link https://mclo.gs/ and also make an dump console command ( geyser dump ). Without any errors we cannot help you out.
Thanks

**Comment by byquanton** [comment]
It's an Issue with the QualityArmory Plugin because when it downloads its dependencies on the first run or updates them, it will reload the Server. More Informationen why you should never "reload" your Minecraft Server: https://madelinemiller.dev/blog/problem-with-reload/.

https://github.com/ZombieStriker/QualityArmory/issues/26
https://github.com/ZombieStriker/QualityArmory/blob/25a62a867237550c4cf1ca2d45d4a471adfaa0a0/src/me/zombie_striker/qg/GithubDependDownloader.java#L85

First Start: (Breaks Geyser)
https://mclo.gs/0sY0XBE
Second Start: (Geyser works fine)
https://mclo.gs/y9kDura

**TL;DR** @shivpatel333 Restart your Server and Geyser should be running fine.

**Comment by Konicai** [comment]
Let us know if this issue still persists with quanton's advice, and provide more info, and this can be re-opened
